app/src/main/java/com/lastaoutdoor/lasta/models/notifications/NotificationDataClasses.kt

```diff
@@ -0,0 +1,5 @@
+package com.lastaoutdoor.lasta.models.notifications
+
+data class SendMessageDto(val to: String?, val notification: NotificationBody)
```

Comment:
You could rename it `SendMessageDTO` because it is a bit confusing with a function that could be named `SendMessageTo`

Reply:
Ok, I'll accept the comment!